log4j-core/src/main/java/org/apache/logging/log4j/core/net/ssl/AbstractKeyStoreConfiguration.java

```diff
@@ -47,21 +48,20 @@ public AbstractKeyStoreConfiguration(
     @Override
     protected KeyStore load() throws StoreConfigurationException {
         final String loadLocation = this.getLocation();
+        final char[] password = this.getPassword();
```

Comment:
I think you can make the `load` method `private`, so there is no way to reload the keystores.

Reply:
Fixed in 4c8d08bf706b6c9e1dedd5a5b7efd3990e474f4f.